apps/playground-sveltekit/.npmrc

```diff
@@ -0,0 +1 @@
+engine-strict=true
```

Comment:
Just curious if this file is necessary, if not, could we drop it to keep the playground minimal?